test/findJava.sh

```diff
@@ -1,6 +1,4 @@
-#!/bin/sh
 bal_command=`which bal`
-bal_command=`realpath "$bal_command"`
```

Comment:
I removed `realpath` since it's not available on macOS.
It works without it on my machine, do we need to find an alternative ?

Reply:
You can just run the make with `JAVA=/foo/bar/bin/java make`, then it shouldn't use findJava.sh.